src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetCohorts/GetCohortsHandler.cs

```diff
@@ -36,6 +36,7 @@ public async Task<GetCohortsResult> Handle(GetCohortsQuery command, Cancellation
                                             {
                                                 AccountId = c.EmployerAccountId,
                                                 LegalEntityName = c.AccountLegalEntity.Name,
+                                                AccountLegalEntityPublicHashedId = c.AccountLegalEntity.PublicHashedId,
```

Comment:
I can't see where this has been used in das-providercommitments PR

Reply:
The Acceptance Criteria was later changed, hence why is not been used but I think is a good thing to have.